mu-plugins/build-processes-demo-features/includes/custom-post-types.php

```diff
@@ -59,3 +59,23 @@ function bpd_features_register_book_post_type(): void {
 	register_post_type( 'book', $args );
 }
 add_action( 'init', 'bpd_features_register_book_post_type' );
+
+/**
+ * Registers and/or enqueues scripts and stylesheets specific to the book post type.
+ *
+ * @since   0.1.0
+ * @version 0.1.0
+ */
+function bpd_features_enqueue_book_post_type_frontend_assets(): void {
+	$plugin_slug = bpd_features_get_slug();
+
+	if ( is_post_type_archive( 'book' ) ) {
+		$asset_meta = bpd_features_get_asset_meta( BPD_FEATURES_DIR . 'assets/css/build/book-archive.css' );
+		wp_enqueue_style( "$plugin_slug-book-archive", BPD_FEATURES_URL . 'assets/css/build/book-archive.css', $asset_meta['dependencies'], $asset_meta['version'] );
```

Comment:
Formatting, these enqueues have a lot going on in one line, could we break this down to params on each new line? This is a personal preference thing and happy for pushback if you feel differently.

Reply:
I don't feel too strongly either way. I tend to use both formats depending on what I think makes the most sense in the context (e.g., if I have an array of multiple dependencies, I most likely go with a multi-row format). 

Let me sleep on this one and get back to you tomorrow 😄